api/tests/unit/application/usecases/checkUserHasRoleCertif_test.js

```diff
@@ -0,0 +1,51 @@
+const { expect, sinon, catchErr } = require('../../../test-helper');
+const useCase = require('../../../../lib/application/usecases/checkUserHasRoleCertif');
+const tokenService = require('../../../../lib/domain/services/token-service');
+const adminMemberRepository = require('../../../../lib/infrastructure/repositories/admin-member-repository');
+const { ForbiddenAccess } = require('../../../../lib/domain/errors');
+
+describe('Unit | Application | Use Case | checkUserHasRoleCertifUseCase', function () {
+  const userId = '1234';
+
+  beforeEach(function () {
+    sinon.stub(tokenService, 'extractUserId').resolves(userId);
+    sinon.stub(adminMemberRepository, 'get');
+  });
+
+  it('should resolve true when the admin member has role certif', function () {
+    // given
+    adminMemberRepository.get.resolves({ isCertif: true });
+
+    // when
+    const promise = useCase.execute(userId);
+
+    // then
+    return promise.then((result) => {
+      expect(result).to.be.true;
+    });
```

Comment:
```suggestion
    const result = await useCase.execute(userId);

    // then
    expect(result).to.be.true;
```

pareil pour les autres tests similaires

Reply:
done